packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-## NEXT
+## 0.8.4+5
 
 * Updates Android compileSdkVersion to 31.
 * Fix iOS RunnerUITests search paths.
+* Fix Activity leak
```

Comment:
Please add the missing period at the end.

Also this should be "Fixes" per the repo's new CHANGELOG style guide. (It would be great if you could update the line above it the same way.)

Reply:
Done.